index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
+</head>
+
+<body>
+    <section class="head">
+      <div class="background-image-1"></div>
+      <ul class="status-bar">
+        <li class="time-text">1:00</li>
+        <li class="group-1">
+          <img class="inside" src="icons/Cellular-Connection.svg" alt="cellular" id="connection">
+          <img class="inside" src="icons/Wifi.svg" alt="Wifi" id="wifi">
+          <img class="inside" src="icons/Battery.svg" alt="battery" id="battery">
+        </li>
+      </ul>
+      <nav class="logo-placeholder">
+      <a href="#" class="logo">Maroun</a>
+      <a href="#"><img src="icons/menu.svg" alt="menu" id="menu"></a>
+      </nav>
+      <div class="headline">
+          <ul>
+        <li class="primary-text">I’m Maroun Glad to see you!</li>
+        <li class="primary-description">I’m a software developer! I can help you build a product , feature or website
+          Look through some of my work and experience! If you like what you see and have a project you need coded, don’t
+          hestiate to contact me.</li>
+        </ul>
+        <a href="#" class="connect">LET’S CONNECT</a>
+
+        <ul class="social-media-icons">
+          <li>
+            <a href="#"><img src="icons/twitter.svg" alt="twitter" id="twitter"></a>
+          </li>
+          <li>
+            <a href="#"><img src="icons/Linkedin.svg" alt="linkedin" id="linkedin"></a>
+          </li>
+          <li>
+            <a href="#"><img src="icons/medium.svg" alt="medium" id="medium"></a>
+          </li>
+          <li>
+            <a href="#"><img src="icons/github.svg" alt="github" id="github"></a>
+          </li>
+          <li>
+            <a href="#"><img src="icons/v-sign.svg" alt="V-sign" id="V-sign"></a>
```

Comment:
This is not displaying as a link on the browser please fix it 🙏

Reply:
It's not working I'm not finding the issue can u plz help
When I move it downwards it displays and acts like a link but when I set it to place it doesn't anymore

Reply:
when i removed position: absolute from  .background-image-1 in your styling it fixed the link issue, maybe you could use flex-box to position the background properly or  check this [Link](https://www.w3schools.com/cssref/pr_background-position.asp)